### ISSUE TITLE
How should the validation set be obtained, the same as the training set?

### ISSUE THREAD

**Issue by VitoZ-X** [raywzy/Bringing-Old-Films-Back-to-Life#29]
The validation set in the profile I'm looking at doesn't seem to be using REDS, but real film video footage, is that correct?
